iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/ReactiveCapabilityCurveUtil.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network.util;
+
+import com.powsybl.iidm.network.ReactiveCapabilityCurve;
+import org.apache.commons.lang3.function.TriFunction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.TreeMap;
+
+/**
+ * @author Sylvestre Prabakaran {@literal <sylvestre.prabakaran at rte-france.com>}
+ */
+public final class ReactiveCapabilityCurveUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ReactiveCapabilityCurveUtil.class);
+
+    private ReactiveCapabilityCurveUtil() {
+    }
+
+    /**
+     * Extrapolate reactive limits when p is outside [minP,maxP] using slopes of reactive limits at the crossed limit of p
+     * (Note that this method throws an exception if p is inside [minP, maxP])
+     * @param p                               Active power value to evaluate the reactive limits
+     * @param points                          TreeMap of all points defining the reactive capability curve
+     * @param valuesToReactiveCapabilityPoint TriFunction returning the used implementation of ReactiveCapabilityCurve.Point interface
+     * @param ownerDescription                Description of the owner (for logging purpose)
```

Comment:
To provide a little more details:

```suggestion
     * @param points                          TreeMap of all points defining the reactive capability curve, mapped by their active power values
     * @param valuesToReactiveCapabilityPoint TriFunction returning the used implementation of {@link ReactiveCapabilityCurve.Point} interface:
     *                                        <code>(p, minQ, maxQ) -> Point</code>
     * @param ownerDescription                Description of the ReactiveCapabilityCurve's owner (for logging purpose)
```

Reply:
Yes it is better inded, done.